src/hotspot/share/prims/jvmti.xml

```diff
@@ -1004,7 +1004,8 @@ jvmtiEnv *jvmti;
     In some cases, <jvmti/> functions allocate memory that your program must
     explicitly deallocate. This is indicated in the individual <jvmti/>
     function descriptions.  Empty lists, arrays, sequences, etc are
-    returned as <code>nullptr</code>.
+    returned as <code>nullptr</code> which is C programming language
+    <code>null</code> pointer.
```

Comment:
Thank you, Kim. I like this suggestion. Updated now.

Reply:
That part looks okay but I think all the parameters and error descriptions changed by JDK-8324680 will now need to change to use "null" instead of "nullptr".

Reply:
Okay. I've made a fix to replace in the doc: `nullptr` => `null` pointer as David suggested below.
I can reduce it and remove the word 'pointer'. Please, let me know what is better.